.circleci/config.yml

```diff
@@ -47,17 +49,29 @@ aliases:
         echo "ruby-${_RUBY_VERSION}" > ~/.ruby-version
         echo 'chruby ruby-${_RUBY_VERSION}' >> ~/.bash_profile
 
-version: 2
+  - &bundle_install
+    run:
+      name: bundle install
+      command: |
+        gem install bundler -v $(cat Gemfile.lock | tail -1 | tr -d " ")
```

Comment:
This way CI can always use `bundler` that has exactly the same version as we used to generate "Gemfile.lock"